File: components/maps/src/main/java/org/storm/maps/tile/Tile.java
package org.storm.maps.tile;

import org.storm.physics.entity.ImmovableEntity;
import org.storm.physics.math.geometry.shapes.AxisAlignedRectangle;

public class Tile extends ImmovableEntity {

  public Tile(double x, double y, double width, double height) {
    super(new AxisAlignedRectangle(x, y, width, height));
  }

}

File: components/maps/src/main/kotlin/org/storm/maps/tile/Tile.kt
package org.storm.maps.tile

import org.storm.physics.entity.ImmovableEntity
import org.storm.physics.math.geometry.shapes.AABB

class Tile(x: Double, y: Double, width: Double, height: Double) : ImmovableEntity(AABB(x, y, width, height))
